### ISSUE TITLE
SemVer 2 versions format

### ISSUE THREAD

**Issue by Sl4vP0weR** [GlitchEnzo/NuGetForUnity#540]
## Description

[Describe the issue you have. If applicable provide logs produced by enabeling verbose logging in the NuGetForUnity settings.]

I guess SemVer 2.0.0 version format is not supported.

```
Compare Error: 1.13.2+0596138b111ff552137684c6f7c3373805d2e3d2 1.13.2+0596138b111ff552137684c6f7c3373805d2e3d2
UnityEngine.Debug:LogErrorFormat (string,object[])
NugetForUnity.NugetPackageIdentifier:CompareVersions (string,string)
NugetForUnity.NugetPackageIdentifier:CompareVersion (string)
NugetForUnity.NugetPackageIdentifier:CompareTo (NugetForUnity.NugetPackageIdentifier)
NugetForUnity.NugetHelper:CheckForUnnecessaryPackages ()
NugetForUnity.NugetHelper:Restore ()
NugetForUnity.OnLoadNuGetPackageRestorer:.cctor ()
UnityEditor.EditorAssemblies:ProcessInitializeOnLoadAttributes (System.Type[])
```

-   **NuGet Package:** [What is the name and version of the NuGet package you are installing?]
[PolySharp](https://www.nuget.org/packages/PolySharp/)
v1.13.2+0596138b111ff552137684c6f7c3373805d2e3d2 

-   **NuGetForUnity Version:** [What version of NuGetForUnity are you running?]
Latest on this moment - 3.1.3

-   **Unity Version:** [What version of Unity are you using?]
2021.3.21f1

-   **Operating System:** [What OS are you on?]
Windows 10 Pro x64

**Comment by popara96** [comment]
How did you manage to find that package? It doesn't seem to show up when I search for it...
![image](https://github.com/GlitchEnzo/NuGetForUnity/assets/31167073/c76f62ee-4ede-487d-a787-252386494d29)

**Comment by Sl4vP0weR** [comment]
Added to the packages.

```xml
<packages>
  <package id="PolySharp" version="1.13.2" />
</packages>
```

**Comment by JoC0de** [comment]
With the fix #543 it's now possible to just install it using the UI
